src/App.js

```diff
@@ -1,12 +1,12 @@
-import React from 'react'
-import { View, Text } from 'react-native'
+import React from 'react';
+import { View, Text, TouchableOpacity } from 'react-native';
 
-const App = () => {
-  return (
-    <View>
-      <Text>App</Text>
-    </View>
-  )
```

Comment:
two component should not be in single file.
please wrap the parent component with redux store.
dont name as App.js, name as index.js inside src